src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -252,6 +263,7 @@ protected BoundExpression CreateUserDefinedConversion(SyntaxNode syntax, BoundEx
                     conversion,
                     @checked: false,
                     explicitCastInCode: isCast,
+                    isExplicitlyNullable: isExplicitlyNullable,
```

Comment:
>isExplicitlyNullable: isExplicitlyNullable, [](start = 20, length = 43)

It doesn't feel right to put isExplicitlyNullable here if this is just an intermediate conversion.